system/jlib/jlog.cpp

```diff
@@ -2185,16 +2185,16 @@ void attachManyLogMsgMonitorsFromPTree(IPropertyTree * tree)
 const LogMsgCategory MCdisaster(MSGAUD_all, MSGCLS_disaster);
 const LogMsgCategory MCuserError(MSGAUD_user, MSGCLS_error);
 const LogMsgCategory MCoperatorError(MSGAUD_operator, MSGCLS_error);
-const LogMsgCategory MCinternalError((LogMsgAudience)(MSGAUD_internal & MSGAUD_programmer), MSGCLS_error, 1);
+const LogMsgCategory MCinternalError(MSGAUD_programmer, MSGCLS_error, 1);
 const LogMsgCategory MCuserWarning(MSGAUD_user, MSGCLS_warning);
 const LogMsgCategory MCoperatorWarning(MSGAUD_operator, MSGCLS_warning);
-const LogMsgCategory MCinternalWarning((LogMsgAudience)(MSGAUD_internal & MSGAUD_programmer), MSGCLS_warning, 1);
+const LogMsgCategory MCinternalWarning(MSGAUD_programmer, MSGCLS_warning, 1);
 const LogMsgCategory MCuserProgress(MSGAUD_user, MSGCLS_progress);
 const LogMsgCategory MCoperatorProgress(MSGAUD_operator, MSGCLS_progress);
 const LogMsgCategory MCdebugProgress(MSGAUD_programmer, MSGCLS_progress);
 const LogMsgCategory MCdebugInfo(MSGAUD_programmer, MSGCLS_information);
-const LogMsgCategory MCstats(MSGAUD_performance, MSGCLS_information);
-const LogMsgCategory MCevent(MSGAUD_monitor, MSGCLS_event);
+const LogMsgCategory MCstats(MSGAUD_programmer, MSGCLS_information);
```

Comment:
From your jira comments I think this should be MSGCLS_progress.  Do we need to distinguish progress from status?  I suspect not.